code/implementation/settings-api/src/main/java/io/cattle/platform/api/settings/manager/SettingsFilter.java

```diff
@@ -36,10 +40,18 @@ public SettingsFilter(List<String> publicSettings, ApiRequest apiRequest) {
     @Override
     public boolean evaluate(Object object) {
         String name = null;
+        List<String> settingsHidden = SETTINGS_HIDDEN.get();
```

Comment:
@ibuildthecloud Using `DynamicStringMapProperty` now for `settings.hidden` instead of `DynamicStringListProperty`